src/tradelocker/tradelocker_api.py

```diff
@@ -1341,3 +1375,156 @@ def modify_order(
         self.log.info(f"Order modification response: {response_json}")
         response_status: str = get_nested_key(response_json, ["s"], str)
         return response_status == "ok"
+    
+    @log_func
+    @tl_typechecked
+    # TODO: this should probably be further expanded / validated
+    def modify_position(
+        self, order_id: int, modification_params: ModificationParamsType
+    ) -> bool:
+        """Modifies a open position.
+
+        Args:
+            order_id (int): Order Id
+            modification_params (_ModificationParamsType): Order modification details
+
+        Returns:
+            bool: True on success, False on error
+        """
+        print('AccessToken: ', self.get_access_token())
```

Comment:
Absolutely remove this - this would be printing user's access token into their logs, which should not be happening :)